modules/Bio/EnsEMBL/Variation/Pipeline/GetCAR/GetCARSeqname.pm

```diff
@@ -99,7 +99,7 @@ sub run {
     my $json_file = $filename . '-car-lu' . '.json';
 
     # Getting the HGVS value from the full file
-    my $cmd = "cut -f6 $hgvs_dir/$hgvs_file > $car_lu_dir/$lu_file";
+    my $cmd = "cut -f6 $hgvs_dir/$hgvs_file | grep -v ] > $car_lu_dir/$lu_file";
```

Comment:
To avoid skipping non redundant ID, I think it could be better skipping just ending with `]`. What you think?
```suggestion
    my $cmd = "cut -f6 $hgvs_dir/$hgvs_file | grep -v ]$ > $car_lu_dir/$lu_file";
```